packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -364,13 +364,23 @@ export function buildRootObject(_powers, _params, testJigSetter = undefined) {
         return invitationP;
       },
       // Shutdown the entire vat and give payouts
-      shutdown: () => {
-        E(zoeInstanceAdmin).shutdown();
+      shutdown: msg => {
+        E(zoeInstanceAdmin).exitAllSeats();
         zcfSeatToZCFSeatAdmin.entries().forEach(([zcfSeat, zcfSeatAdmin]) => {
           if (!zcfSeat.hasExited()) {
             zcfSeatAdmin.updateHasExited();
           }
         });
+        powers.exitVat(msg);
+      },
```

Comment:
```suggestion
      shutdown: completion => {
        E(zoeInstanceAdmin).exitAllSeats(completion);
        zcfSeatToZCFSeatAdmin.entries().forEach(([zcfSeat, zcfSeatAdmin]) => {
          if (!zcfSeat.hasExited()) {
            zcfSeatAdmin.updateHasExited();
          }
        });
        powers.exitVat(completion);
      },
```

Reply:
> There's no completion parameter when exiting seats.

I understand that. I'm saying there should be, for uniformity with the rest of the multi-level terminating architecture. Given the rest of this PR, I think this PR would be an appropriate time. For example, the seat's completion should probably also become the completion of the seat's notifier.

Reply:
I'm convinced.